config/load.go

```diff
@@ -161,6 +162,23 @@ func (c *Config) checkAndHotReloadConfig(configMap map[string][]*configValue) {
 					fmt.Printf("The value of key:%s & variable:%p changed from %v to %v\n", key, configVal, *value, _value)
 					*value = _value
 				}
+			case *[]string:
+				var _value []string
+				var isSet bool
+				for _, key := range configVal.keys {
+					if c.IsSet(key) {
+						isSet = true
+						_value = c.GetStringSlice(key, configVal.defaultValue.([]string))
+						break
+					}
+				}
+				if !isSet {
+					_value = configVal.defaultValue.([]string)
+				}
+				if !reflect.DeepEqual(_value, *value) {
```

Comment:
Should we do a normal []string comparison instead of reflect?

Reply:
Will this work?
`slices.Compare(_value, *value)`

Reply:
Using `slices.Compare(_value, *value) != 0` now.